src/main/java/core/basesyntax/Entry.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Entry<K, V> {
+    private final K key;
+    private V value;
+
+    public Entry(K key, V value) {
+        this.key = key;
+        this.value = value;
+    }
+
+    public V getValue() {
+        return value;
+    }
+
+    public void setValue(V value) {
+        this.value = value;
+    }
+
+    public K getKey() {
+        return key;
+    }
+
+}
```

Comment:
According to the checklist, the `Entry` class should not have getters and setters as we have access to the private fields of the inner class. You should remove `getValue`, `setValue`, and `getKey` methods.